pkgs/top-level/all-packages.nix

```diff
@@ -8006,15 +8006,13 @@ with pkgs;
 
   chewing-editor = libsForQt5.callPackage ../applications/misc/chewing-editor { };
 
+  mozc = qt6Packages.callPackage ../development/libraries/mozc { };
+
   fcitx5 = callPackage ../tools/inputmethods/fcitx5 { };
 
   fcitx5-bamboo = callPackage ../tools/inputmethods/fcitx5/fcitx5-bamboo.nix { };
 
-  fcitx5-mozc = libsForQt5.callPackage ../tools/inputmethods/fcitx5/fcitx5-mozc.nix {
-    abseil-cpp = abseil-cpp.override {
-      cxxStandard = "17";
-    };
-  };
+  fcitx5-mozc = callPackage ../tools/inputmethods/fcitx5/fcitx5-mozc.nix { };
```

Comment:
I think the by-name check won't allow this anymore, it will probably trigger on the next commit. Do you think you could move it to `pkgs/by-name`?